doc/conf.py

```diff
@@ -19,37 +20,41 @@
 source_suffix = ".rst"
 
 root_doc = "index"
-master_doc = "index"  # deprecated in 4.0
 
-project = "Qserv"
-
-# TODO
-copyright = ""
-
-author = "Rubin Observatory"
+# General information about the project.
+project = u'Qserv'
+copyright = u'2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)'
+author = u'LSST Data Management'
```

Comment:
AFAIK python3 strings are unicode by default, the leading `u` is left in the language for portability with python2. It probably should be removed here.

Reply:
will do